### PR TITLE
fix structured result

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

File: src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java [hunk 1]
```diff
@@ -46,6 +46,7 @@
 @Repository
 public class SensitivityAnalysisResultRepository {
 
+    public static final double MINIMUM_SENSITIVITY = 0.01;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java:49]: not used

File: src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java [hunk 2]
```diff
@@ -136,7 +137,9 @@ private int apply(AnalysisResultEntity sas, Collection<String> funcIds, Collecti
 
             int ci = sar.getContingencyIndex();
             ContingencyEmbeddable c = ci < 0 ? null : cs.get(ci);
-            if ((ci < 0) == beforeOverAfter) {
+            if ((ci < 0) != (allContingencyIds == null)) {
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java:140]: checkstyle UnnnecessaryParentheses is raised here (with powsybl checkstyle configuration file)

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java (hunk 1 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java:49]: You're right. I remove.

File: src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java (hunk 2 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java:140]: Uh, wouldn't have guessed, as it is not my habit of using surnumeral parentheses.
But here I wanted to emphasize that the topmost condition is an equality between booleans,
so I addes parentheses around the inequality test.

File: src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java (hunk 1 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java]: Done

File: src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java (hunk 2 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java]: Done

**Review by FranckLecuyer — APPROVED** [review]
Code review: OK
Tests: OK

**Pull request merged**